client/my-sites/patterns/components/readymade-template-details/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { addLocaleToPathLocaleInFront } from '@automattic/i18n-utils';
+import { Button } from '@wordpress/components';
+import { Icon, arrowLeft, check, copy } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect, useState } from 'react';
+import { PatternsGetStarted } from 'calypso/my-sites/patterns/components/get-started';
+import { useReadymadeTemplates } from 'calypso/my-sites/patterns/hooks/use-readymade-templates';
+import { ReadymadeTemplateDetailsFC } from 'calypso/my-sites/patterns/types';
+
+import './style.scss';
+
+export const ReadymadeTemplateDetails: ReadymadeTemplateDetailsFC = ( { id } ) => {
+	const translate = useTranslate();
+	const [ isCopied, setIsCopied ] = useState( false );
+
+	useEffect( () => {
+		window.scroll( 0, 0 );
+	}, [] );
+
+	const { data: readymadeTemplates = [] } = useReadymadeTemplates();
+	if ( ! readymadeTemplates.length ) {
+		return null;
+	}
+
+	const readymadeTemplate = readymadeTemplates.find( ( rt ) => rt.template_id === id );
+	if ( ! readymadeTemplate ) {
+		return null;
+	}
+
+	const copyLayout = () => {
+		navigator.clipboard.writeText( readymadeTemplate.content );
+		setIsCopied( true );
+	};
+
+	return (
+		<>
+			<section className="readymade-template-details-wrapper">
+				<div className="readymade-template-details-section">
+					<a
+						href={ addLocaleToPathLocaleInFront( '/patterns' ) }
+						className="readymade-template-details-back"
+					>
+						<Icon icon={ arrowLeft } size={ 24 } /> { translate( 'Back to patterns' ) }
```

Comment:
Unsure about this, and whether people will recognise that "Back to patterns" is a chance to choose a different "site layout", particularly as going back scrolls to the top of the page.

Maybe just saying 'Back' or 'Back to patterns and layouts' would help.

Reply:
Changed it to just "Back": https://github.com/Automattic/wp-calypso/pull/92078/commits/ade6a8072baa42508890fcbee7b8c9c1e5b5b638